src/courtroom.cpp

```diff
@@ -4396,7 +4396,7 @@ void Courtroom::on_reload_theme_clicked()
   text_state = 3;
 
   // to update status on the background
-  set_background(current_background);
+  set_background(current_background, true);
```

Comment:
preemptive explanation: set_background second variable is for "display", which dictates that the background will be displayed as soon as it is set. This way, the displayed background is done according to the current viweport scale, and since this function is called *after* the new theme settings are set, + it clears all the characters from view, it behaves as expected and properly resets all the sizes that need to be set.

Reply:
ok lobo